selfupdate-docker/test/test-tedge-updater.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+
+#
+# Module-Test tedge_updater
+#
+
+CNOTE='\033[0;33m'
+CINFO='\033[0;34m' # blue
+CERROR='\033[0;31m'
+CNORMAL='\033[0m' # No Color
+
+# Args:
+#   $1 - optional set to "force-wait" to wait for zusu user-key despite of option "-w"
+wait_for_key() {
+   if [ "$DO_WAIT" = "YES" ] || [ "$1" = "force-wait" ]; then
+     echo -e "${CNOTE}Press any key to continue${CNORMAL}"
+     read -n 1
+     echo -e "${CNOTE}Thanks, going on..${CNORMAL}"
+   fi
+}
+
+check_for_cmd() {
+   if ! command -v $1 &> /dev/null
+   then
+      echo -e "${CERROR}ERROR: Command $1 could not be found"
+      exit
+   fi
+}
+
+# Args:
+#   $1 - text to display
+#   $2 - optional set to "force-wait" when wait for user-key despite of option "-w" 
+test_info() {
+   echo -e "${CINFO}INFO: $1${CNORMAL}"
+   wait_for_key $2
+}
+
+
+echo "Check for required commands"
+check_for_cmd read
+check_for_cmd pushd
+
+DO_BUILD="YES"
+DO_WAIT="YES"
+DO_UPLOAD_CERT="YES"
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -b|--no-build)
+      DO_BUILD="NO"
+      echo "Disabled build by cmdline argument."
+      shift # past argument
+      ;;
+    -w|--no-wait)
+      DO_WAIT="NO"
+      echo "Disabled user interaction."
+      shift
+      ;;
+     -c|--no-cert-upload)
+      DO_UPLOAD_CERT="NO"
+      echo "Disabled upload of certificate to cloud."
+      shift
+      ;;
+     --fail-container-start)
+      echo "Forcing new container start to fail (to test rollback)"
+      DO_CONTAINER_START_FAIL="YES"
+      shift
+      ;;
+     -*|--*)
+      echo "Unknown option $1"
+      exit 1
+      ;;
+    *)
+      echo "Unknown option $1"
+      exit 1
+      ;;
+  esac
+done
+
+#
+# (1) Building Docker Image
+#
+
+pushd ../docker-image/ > /dev/null
```

Comment:
Apply change of folder name.

```suggestion
pushd ../docker/ > /dev/null
```